FizzBuzz 1/FizzBuzz/Program.cs

```diff
@@ -1,42 +1,117 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace FizzBuzz
 {
     class Program
     {
+        static Boolean ReadYN(string message)
```

Comment:
They are the same so not very important but `bool` is slightly more widely used 🙂